src/features/applySubstitution.js

```diff
@@ -1,3 +1,11 @@
+/*
```

Comment:
Can you please remove these file headers?

Reply:
oh, of course